tests/harness/executor/BUILD

```diff
@@ -12,8 +12,8 @@ go_library(
     visibility = ["//visibility:private"],
     deps = [
         "//tests/harness:go_default_library",
-        "//tests/harness/cases:go",
```

Comment:
These changes should be reverted. If you look at `tests/harness/cases/BUILD`, you'll see that it declares a target named "go", not "go_default_library". For future reference, `//tests/harness/cases/go:go_default_library` would be defined in the file `tests/harness/cases/go/BUILD`.